stablehlo/reference/Tensor.h

```diff
@@ -61,7 +61,8 @@ class Buffer : public llvm::RefCountedBase<Buffer> {
 
 }  // namespace detail
 
-/// Helper class to access the tensor elements in a linearized layout.
+/// Class to model a tensor, an n-dimensional array. Provide access to
+/// individual elements of the tensor using n-dimensional indices.
```

Comment:
This is missing some important aspects (lifetime, mutability, etc.)

Reply:
I agree. Let is take this up in  https://github.com/openxla/stablehlo/pull/117#issuecomment-1262869759